internal/nodes/avi_model_l7_hostname_shard.go

```diff
@@ -525,7 +526,7 @@ func (vsNode *AviVsNode) AddSSLPort(key string) {
 }
 func (vsNode *AviVsNode) DeletSSLRefInDedicatedNode(key string) {
 	vsNode.SSLKeyCertRefs = []*AviTLSKeyCertNode{}
-	vsNode.SSLProfileRef = ""
+	vsNode.SslProfileRef = proto.String("")
```

Comment:
This may cause problem from ConvertToRef point of view, since pointer to sslprofileref will not be nil.

Reply:
I have addressed the comment. Thanks